Src/Extern/SENSEI/AMReX_AmrDataAdaptor.cpp

```diff
@@ -605,6 +605,8 @@ int AmrDataAdaptor::GetArrayName(const std::string &meshName,
 int AmrDataAdaptor::GetMesh(const std::string &meshName,
     bool structureOnly, vtkDataObject *&mesh)
 {
+    (void) structureOnly;
```

Comment:
```suggestion
    amrex::ignore_unused(structureOnly);
```

This is maybe slightly more clear that the intent is to silence unused variable warnings.